LHM/Problems/beakjoon/10818.py

```diff
@@ -0,0 +1,20 @@
+#https://www.acmicpc.net/problem/10818
+N = int(input())
+print(N)
+input_val =[]
+
+
+for i in range(0,N):
+    input_val.append(input())
```

Comment:
백준 문제에서 언급된 input 형태가 아닌거 같습니다.
하나씩 중첩하는 것이 아닌 한번에 List Array를 주는 것을 Input으로 정의하고 있어 보입니다.

아래 내용도 한번 검토해주세요!

```suggestion
    input_val = list(map(int,input().split()))
```

Reply:
아니면 list의 extend 함수 사용해서 input을 list로 담는 방법도 괜찮을 듯?